src/welldev/io/controller/LogOut.java

```diff
@@ -0,0 +1,29 @@
+package welldev.io.controller;
+
+import java.io.IOException;
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+import welldev.io.utility.DBConnection;
+
+
+@WebServlet("/logout")
+public class LogOut extends HttpServlet {
+	
+	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+		DBConnection dbConnection = new DBConnection();
+		dbConnection.closeConnection();
```

Comment:
the database connection must be closed after every database transaction. you dont need to close it here. Beacuse HTTP is stateless.